### PR TITLE
Outlook cheetsheat

### PR DESCRIPTION
**Pull request by hazeeb**
Outlook cheetsheat.

//cc @hemanth 

---
https://duck.co/ia/view/outlook_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@hazeeb Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: t/IsAwesome/hazeeb.t [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env perl
```
Comment (MariagraziaAlastra) [inline t/IsAwesome/hazeeb.t]: @hazeeb Looks like this was included by mistake? Can you remove this file, please?

File: lib/DDG/Goodie/IsAwesome/hazeeb.pm [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package DDG::Goodie::IsAwesome::hazeeb;
```
Comment (MariagraziaAlastra) [inline lib/DDG/Goodie/IsAwesome/hazeeb.pm]: @hazeeb Same here: can you please remove this file?

**Comment by MariagraziaAlastra** [comment]
Hey @hazeeb, thanks for your contribution, I just left a couple of comments in the code.
Looks like you're developing from your master branch, which can lead to this sort of issues - different PRs will show all the files you've been working on; it would be much easier to separate your work into different branches instead :smile: 
You can find a guide to the suggested git workflow [in the docs](http://docs.duckduckhack.com/resources/git-workflow.html).

Also, did you create an IA Page for your Cheat Sheet? If not, please do so [here](https://duck.co/ia/new_ia) and then paste the link to it in your first comment.
[The docs](http://docs.duckduckhack.com/submitting/pull-request.html#submitting-a-pull-request) also contain some useful info about submitting pull requests.
Thanks!

**Comment by hemanth** [comment]
@MariagraziaAlastra Thanks for the detailed info. :+1:

File: share/goodie/cheat_sheets/json/outlook.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            "key": "[shift], [I]",
+            "val": "Show blocked content"
+        }, {
+            "key": "[/]",
+            "val": "Search your email"
+        }, {
+            "key": "[Esc]",
+            "val": "Close an email"
+        }, {
+            "key": "[X]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/outlook.json]: @hazeeb when dealing with a single keyboard key, you shouldn't wrap it in `[]` they're just used to separate multiple keys!

Comment (moollaza) [inline share/goodie/cheat_sheets/json/outlook.json]: e.g. `"key": "X"`

File: lib/DDG/Goodie/IsAwesome/hazeeb.pm (hunk 2 above)
Comment (hazeeb) [inline lib/DDG/Goodie/IsAwesome/hazeeb.pm]: Sure. I've deleted this file.

File: t/IsAwesome/hazeeb.t (hunk 1 above)
Comment (hazeeb) [inline t/IsAwesome/hazeeb.t]: Done.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/outlook_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/outlook.json [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    "section_order": [
+        "Write email",
+        "Read email",
+        "Go to",
+        "Email actions",
+        "Email list"
+    ],
+    "sections": {
+        "Write email": [{
+            "key": "N / [Ctrl] [N]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/outlook.json]: whoops, don't forget that first `N`: `"key": "[N] / [Ctrl] [N]"`

File: share/goodie/cheat_sheets/json/outlook.json [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+            "val": "Insert an Emoji"
+        }, {
+            "key": "[Ctrl] [S]",
+            "val": "Save draft"
+        }],
+        "Read email": [{
+            "key": "[O] / [Enter]",
+            "val": "Open an email"
+        }, {
+            "key": "[shift], [I]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/outlook.json]: Let's capitalize `Shift` to match the other ones here

File: share/goodie/cheat_sheets/json/outlook.json [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+            "key": "[S], [N]",
+            "val": "Deselect all emails"
+        }, {
+            "key": "[Ctrl] [.] / Down",
+            "val": "Move selection downwards"
+        }, {
+            "key": "[Ctrl] [.] / Up",
+            "val": "Move selection upwards"
+        }, {
+            "key": "M / F9",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/outlook.json]: Don’t forget to wrap these keys in `[ ]`.

Typically for alternatives we suggest you create a new key-value pair and indicate that its an alternative.

File: share/goodie/cheat_sheets/json/outlook.json [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+            "val": "Print an email"
+        }],
+        "Email list": [{
+            "key": "[S], [A]",
+            "val": "Select all emails"
+        }, {
+            "key": "[S], [N]",
+            "val": "Deselect all emails"
+        }, {
+            "key": "[Ctrl] [.] / Down",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/outlook.json]: Is `Down` the Down Arrow? If so please use the Ascii Arrows. You can find them in the docs:

http://docs.duckduckhack.com/frontend-reference/cheat-sheet-reference.html#arrow-keys

File: share/goodie/cheat_sheets/json/outlook.json [hunk 8]
```diff
@@ -0,0 +1,135 @@
+{
+    "id": "outlook_cheat_sheet",
+    "name": "outlook",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/outlook.json]: nit: We should capitalize `Outlook`.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=outlook cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/outlook_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=outlook cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/outlook_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=outlook cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/outlook_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=outlook cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/outlook_cheat_sheet_mobile.png)

File: lib/DDG/Goodie/IsAwesome/hazeeb.pm (hunk 2 above)
Comment (moollaza) [inline lib/DDG/Goodie/IsAwesome/hazeeb.pm]: @hazeeb you need to make sure you `commit` the deletion and `push` the update here so the file is no longer a part of your Pull Request :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=outlook cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/outlook_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=outlook cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/outlook_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=outlook cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/outlook_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=outlook cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/outlook_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=outlook cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/outlook_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/outlook.json [hunk 9]
```diff
@@ -0,0 +1,135 @@
+{
+    "id": "outlook_cheat_sheet",
+    "name": "Microsoft Outlook",
+    "description": "“Overview of common commands for Microsoft Outlook",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/outlook.json]: nit: you have a leading `"` that should be removed

**Comment by moollaza** [comment]
@hazeeb this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

If you need ideas about how to contribute next, please feel free to reach out to me on Slack! I'm happy to chat about how you can help :+1:

**Pull request merged**